packages/components/src/Drawer/Drawer.test.tsx

```diff
@@ -4,45 +4,57 @@ import { Drawer } from ".";
 
 afterEach(cleanup);
 
-test("drawer shows the children and a close button", () => {
-  const title = "Drawer title";
-  const content = "Drawer content";
-  const handleClose = jest.fn();
-
-  const { getByLabelText, getByText, queryByTestId } = render(
-    <Drawer title={title} open={true} onRequestClose={handleClose}>
-      {content}
-    </Drawer>,
-  );
-
-  expect(queryByTestId("drawer-header")).not.toBeNull();
-  expect(getByText(title)).toBeTruthy();
-  expect(getByLabelText("Close drawer")).toBeTruthy();
-  expect(getByText(content)).toBeTruthy();
-});
-
-it("doesn't render a closed drawer", () => {
-  const handleClose = jest.fn();
-
-  const { queryByTestId } = render(
-    <Drawer title="Drawer" open={false} onRequestClose={handleClose}>
-      Drawer content
-    </Drawer>,
-  );
-  expect(
-    queryByTestId("drawer-container").classList.contains("open"),
-  ).toBeFalsy();
-});
-
-test("drawer fires onRequestClose when selecting the close button", () => {
-  const handleClose = jest.fn();
-
-  const { getByLabelText } = render(
-    <Drawer title="Drawer" onRequestClose={handleClose}>
-      Drawer content
-    </Drawer>,
-  );
-
-  fireEvent.click(getByLabelText("Close drawer"));
-  expect(handleClose).toHaveBeenCalledTimes(1);
+describe("Drawer", () => {
+  describe("when open", () => {
+    it("should render children and close button", () => {
+      const content = "Drawer Content";
+      const { container, getByText } = render(
+        <Drawer title="A drawer with content" open onRequestClose={jest.fn}>
+          {content}
+        </Drawer>,
+      );
+      expect(getByText(content)).toBeInTheDocument();
+      expect(container).toMatchSnapshot();
```

Comment:
Until we've got a decision re: snapshot tests, can we agree to keep what's there and potentially do a fast-follow to update these tests?

Reply:
That's fair, tho I'm pretty sure it'll take you longer to create the branch, come up with the title, write the description ,do the change than splitting this test into 2. Or better yet, revert to the old test as that's much more secure than us (humans) manually scanning the snapshot.

```
  expect(queryByTestId("drawer-header")).not.toBeNull();
  expect(getByText(title)).toBeTruthy();
  expect(getByLabelText("Close drawer")).toBeTruthy();
  expect(getByText(content)).toBeTruthy();
```
_But maybe use the `toBeInTheDocument`_

Also, this PR is not done yet as per your description _"As the drawer is similar to the Card"_. So if we're basing the decision for this component to not have a default padding because the card didn't have any, said padding should be brought back by the `<Content />` component when used inside this component. Just like the card.

#### Card

Note: the word content here pertains to the `<Content />` component.

![Card___🔱_Atlantis_and_Card___🔱_Atlantis_and__10__Delete_a_Friend___Jobber_Docs](https://user-images.githubusercontent.com/15986172/136052102-8c6ccf93-10e6-4779-bf22-684510ce89d0.png)

#### Modal

_For good measure_

![Modal___🔱_Atlantis_and_Modal___🔱_Atlantis_and_Drawer___🔱_Atlantis](https://user-images.githubusercontent.com/15986172/136052411-54ab8a47-2466-4c64-a830-7a9e39ce7551.png)

#### Drawer (This PR)
![Drawer___🔱_Atlantis_and_Drawer___🔱_Atlantis_and_Drawer___🔱_Atlantis](https://user-images.githubusercontent.com/15986172/136055118-af41e426-c828-4e00-bfc5-8bbf5910d039.png)

Reply:
@darryltec the one thing I noticed that's weird with the drawer and I'm not sure if this is intended, however these tests fail unexpectedly, as the drawer is never actually hidden (either through aria attributes or actually hidden with conditional rendering).

It's just hidden with the lack of css class `open`, which makes the tests you're suggesting difficult.

Should that be tackled here?

Reply:
Wait. I'm only suggesting to revert this 1 `it` back to the previous one

```js
    it("should render the drawer", () => {
      const title = "A drawer with content";
      const content = "Drawer Content";
      const { getByText, getByTestId, getByLabelText } = render(
        <Drawer title={title} open onRequestClose={jest.fn}>
          {content}
        </Drawer>,
      );
      expect(getByTestId("drawer-header")).toBeInTheDocument();
      expect(getByText(title)).toBeInTheDocument();
      expect(getByLabelText("Close drawer")).toBeInTheDocument();
      expect(getByText(content)).toBeInTheDocument();
    });
```

I'm curious now, what do you think i'm suggesting? 😛

Reply:
![image](https://user-images.githubusercontent.com/80279872/136092679-44033350-8532-4688-981e-c9cb7264c727.png)

resolved content padding in 7e53ea0

resolved test case in 1eb2f37